core/src/main/java/com/capitalone/dashboard/model/ServiceStatus.java

```diff
@@ -4,14 +4,36 @@
  * Denotes the status of a product or service
  */
 public enum ServiceStatus {
-    Ok, Warning, Alert;
+    Ok(200, 200),
+    Warning(300, 400),
+    Unauth(401, 401),
```

Comment:
@nireeshT If we have a status for 403, this insinuates that credentials are being passed to the monitored service. At this time, no credentials are being passed, so 403 was not included.